src/status_im2/contexts/chat/messages/content/text/view.cljs

```diff
@@ -40,14 +40,19 @@
     :mention
     (conj
      units
-     [rn/touchable-opacity
-      {:active-opacity 1
-       :on-press       #(rf/dispatch [:chat.ui/show-profile literal])
-       :style          (merge style/block {:background-color colors/primary-50-opa-10})}
-      [quo/text
-       {:weight :medium
-        :style  {:color (colors/theme-colors colors/primary-50 colors/primary-60)}}
-       (rf/sub [:messages/resolve-mention literal])]])
+     [quo/text
+      {:weight                :medium
+       :on-press              #(rf/dispatch [:chat.ui/show-profile literal])
+       :selection-color       :transparent
+       :suppress-highlighting true
+       :style                 (merge style/block
+                                     {:color            (colors/theme-colors colors/primary-50
+                                                                             colors/primary-60)
+                                      :background-color colors/primary-50-opa-10})}
+      " " ;; Text inside Text doesn't support padding, so we need to add a space on each side to mimic
```

Comment:
consider making a helper function outside this scope and leave the comment there.
e.g 

```
 ;; Text inside Text doesn't support padding, so we need to add a space on each side to mimic
;; that effect
(defn pad-text [s]
(str " " s " ")
```

Reply:
I'm not sure if we will need to reuse this. However, if you think it's best to create a helper function for this, is there a common place we keep this kind of helpers?